frontend/css/playlists.less

```diff
@@ -71,6 +71,18 @@
   }
 }
 
+.import-playlist-btn {
+  position: relative;
+  align-self: center;
+  margin-left: auto;
+}
+
+.playlist-header {
+  margin-top: 1em;
+  display: flex;
+  justify-content: space-between;
+}
+
```

Comment:
Probably don't need this class.
Regarding the import button placement, the only thing you need is the `tertiary-nav` css class on the parent component on [line 136](https://github.com/metabrainz/listenbrainz-server/pull/2800/files#diff-50740f3f836f659fc01031697d7a52c784011746a687be5433890665a4d76cb1R136) and you can remove the `style={{ marginTop: "1em" }}`.
Result:
![image](https://github.com/metabrainz/listenbrainz-server/assets/6179856/9e0dfc47-47a4-463d-a7da-513b17da810f)